Changes

```diff
@@ -393,6 +393,11 @@ Working version
 
 ### Internal/compiler-libs changes:
 
+- #13839: Reimplement `let open`, `let module` and `let exception` in terms of a
+  single construct.
+  (Nicolás Ojeda Bär, review by Gabriel Scherer, Samuel Vivien and Vincent
+  Laviron)
```

Comment:
Maybe also add @voodoos as he did the first pass on the code ?

Reply:
Of course, thanks for the reminder. Fixed.